src/atoms/cc-toggle.js

```diff
@@ -286,7 +288,7 @@ export class CcToggle extends LitElement {
         /* NOT SELECTED */
         label {
           --color-bg: #fff;
-          --color-txt: #666;
+          --color-txt: var(--color-text-normal);
```

Comment:
could be set to `--color-text-light` (#404040) if you want to be closer to what we had before.

Reply:
It's OK :+1: